.claude/skills/ux-psychology.md

```
@@ -0,0 +1,446 @@
+# UX心理学ガイドライン
+
+このスキルは、UXデザインに活用できる心理学の法則と原則を提供します。
+
+参考: https://www.shokasonjuku.com/ux-psychology
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)